i18n/ja.toml

```diff
@@ -5,7 +5,7 @@
 other = "始めましょう"
 
 [action_get_started_link]
-other = "/docs/intro/quickstart"
+other = "/ja/docs/intro/quickstart"
```

Comment:
Are you sure you want to link to the Japanese here?

Reply:
Yes, it's no bother.